src/components/StudyForm/StudyForm.tsx

```diff
@@ -0,0 +1,211 @@
+              onIonInput={field.onChange}
+              {...field}
+            />
+          )}
+        />
+
+        <Controller
+          name="metadata_submission.studyForm.notes"
+          control={control}
+          render={({ field, fieldState }) => (
+            <IonTextarea
+              className={`${(fieldState.isTouched || formState.isSubmitted) && "ion-touched"} ${fieldState.invalid && "ion-invalid"}`}
+              label="Notes"
+              helperText="Add any additional notes or comments about this study."
+              labelPlacement="floating"
+              autoGrow
+              onIonBlur={field.onBlur}
+              onIonInput={field.onChange}
+              {...field}
+            />
+          )}
+        />
+      </div>
+
+      <SectionHeader>Principal Investigator</SectionHeader>
+
+      <div className="ion-padding">
+        <Controller
+          name="metadata_submission.studyForm.piName"
+          control={control}
+          render={({ field, fieldState }) => (
+            <IonInput
+              className={`${(fieldState.isTouched || formState.isSubmitted) && "ion-touched"} ${fieldState.invalid && "ion-invalid"}`}
+              label="Name"
+              helperText="The Principal Investigator who led the study and/or generated the data."
+              labelPlacement="floating"
+              type="text"
+              onIonBlur={field.onBlur}
+              onIonInput={field.onChange}
+              {...field}
+            />
+          )}
+        />
+
+        <Controller
+          name="metadata_submission.studyForm.piEmail"
+          control={control}
+          rules={{
+            required: "This field is required",
+            validate: {
+              validEmail: (value: string) =>
+                EMAIL_REGEX.test(value) || "Not a valid email address",
+            },
+          }}
+          render={({ field, fieldState }) => (
+            <IonInput
+              className={`${(fieldState.isTouched || formState.isSubmitted) && "ion-touched"} ${fieldState.invalid && "ion-invalid"}`}
+              helperText="An email address for the Principal Investigator."
+              errorText={fieldState.error?.message}
+              labelPlacement="floating"
+              type="text"
+              onIonBlur={field.onBlur}
+              onIonInput={field.onChange}
+              {...field}
+            >
+              <div slot="label">
+                Email<IonText color="danger">*</IonText>
+              </div>
+            </IonInput>
+          )}
+        />
+
+        <Controller
+          name="metadata_submission.studyForm.piOrcid"
+          control={control}
+          render={({ field, fieldState }) => (
+            <IonInput
+              className={`${(fieldState.isTouched || formState.isSubmitted) && "ion-touched"} ${fieldState.invalid && "ion-invalid"}`}
+              label="ORCID iD"
+              helperText="ORCID iD of the Principal Investigator.."
```

Comment:
Suggestion:

Three dots:

```diff
- helperText="ORCID iD of the Principal Investigator.."
+ helperText="ORCID iD of the Principal Investigator..."
```

Or no dots:

```diff
- helperText="ORCID iD of the Principal Investigator.."
+ helperText="ORCID iD of the Principal Investigator"
```

Reply:
It was supposed to be one period 😂. Bad copy-and-pasting from the submission portal.